app/templates/src/main.html

```diff
@@ -97,5 +97,16 @@ <h3>Statistics</h3>
                 </footer>
             </div>
         </div>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-101646199-7"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'UA-101646199-7');
```

Comment:
Does this need to be configurable or is this page only ever used on Prod?

Reply:
It would only count as a view from prod, because it will only activate in the right domain